server/middleware/setUpDomainRedirect.test.ts

```diff
@@ -0,0 +1,46 @@
+import express, { Express } from 'express'
+import request from 'supertest'
+import setUpDomainRedirect from './setUpDomainRedirect'
+import config from '../config'
+
+jest.mock('../config', () => ({
+  secondDomain: 'http://example.com',
+  firstDomain: 'http://example.org',
+}))
+
+const setupApp = (): Express => {
+  const app = express()
+  app.use(setUpDomainRedirect())
+
+  app.get('/known', (_req, res, _next) => {
+    res.send('known')
+  })
+
+  return app
+}
+describe('setUpDomainRedirect', () => {
+  it('should redirect to the target domain when in test mode', () => {
+    process.env.NODE_ENV = 'test'
+    const app = setupApp()
+    const targetHost = new URL(config.firstDomain).host
+    return request(app).get('/known').set('host', targetHost).expect(301)
```

Comment:
[suggestion] I know this was copied from the other test, but I don't think we need the `return` statement in these tests.